riot/runtime/util/fd.ml

```diff
@@ -54,6 +54,11 @@ let use ~op_name t fn =
   | Some sock -> fn sock
   | None -> raise (Already_closed (op_name ^ ": fd already closed"))
 
+let seek t pos command =
+  match get t with
```

Comment:
most of the stuff inside the `runtime` folder is not really suitable to be used outside. its either not doing the right checks or failing with uncatcheable exceptions.

in this case the `get` function isn't really used anywhere but in `runtime/io/io.ml` and we should probably remove this from the riot.mi

i think we can implement this higher-level function in the lib/fd.ml so we can make it friendlier to use, and we can use the `use` function instead to make sure we only ever seek on a file descriptor that is still open.

Reply:
Ok, just to see if I got it right:

1. I've changed the usage from `get` to `use`;
2. I'm not sure about the `runtime` part because `Fd.ml` includes the Runtime.Util.Fd`
3. I personally prefer interacting with `seek` through `File`, so I'm not sure I think that we should expose it through `Fd`